src/v3/src/util/theme.ts

```diff
@@ -153,22 +154,42 @@ export const createThemeAndTokens = (
       },
       MuiInputBase: {
         styleOverrides: {
-          root: {
+          root: ({ ownerState }) => ({
             width: '100%',
-          },
+            // Odyssey sets flex: "1" but that results in the following IE11 flexbug
+            // https://github.com/philipwalton/flexbugs?tab=readme-ov-file#flexbug-7
+            flex: 'auto',
```

Comment:
This flexbug fix should be in other migrations that use `InputBase` but it can go in with this one